config/localization/en/tooltips.json

```diff
@@ -198,6 +198,9 @@
     "UNREALIZED_PNL": {
       "TITLE": "Unrealized Profit / Loss",
       "BODY": "Total profit / loss if you were to fully close the remainder of your current position at the current oracle price. Resets if you close your position or change position sides (long to short, or vice versa)."
+    },
+    "REWARD_HISTORY": {
+        "BODY": "Time is displayed in Coordinated Universal Time (UTC)"
```

Comment:
note, the tooltip does not have a title per [design](https://www.figma.com/file/fMQodZKzn9B5aZTN5G0fLJ/dYdX-%E2%80%BA-Desktop?type=design&node-id=21144-51979&mode=design&t=7MbRGKHdFcSRqMUS-4)